package.json

```diff
@@ -6,9 +6,7 @@
   "directories": {
     "doc": "docs"
   },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "scripts": {},
```

Comment:
Maybe we should replace the original line "echo ...." with the new line to execute the tests: "npx playwright test"

I think, if we do that, then users can enter "npm test" and it will automatically lookup the "real" command in this file, and run "npx playwright test"